config.ru

```diff
@@ -1,6 +1,6 @@
 require "rack-timeout"
 use Rack::Timeout           # Call as early as possible so rack-timeout runs before other middleware.
-Rack::Timeout.timeout = 20
+Rack::Timeout.timeout = 6
```

Comment:
i think 5 is fine.  In practice, it's still gonna exceed the timeout set on the lms' client, because of network latency.